package.json

```
@@ -1,24 +1,24 @@
 {
   "name": "fastify",
-  "version": "2.3.0",
+  "version": "2.10.0",
   "description": "Fast and low overhead web framework, for Node.js",
   "main": "fastify.js",
   "types": "fastify.d.ts",
   "scripts": {
     "lint": "npm run lint:standard && npm run lint:typescript",
     "lint:standard": "standard --verbose | snazzy",
-    "lint:typescript": "eslint -c types/.eslintrc.json types/**/*.d.ts test/types/**/*.test-d.ts",
-    "unit": "tap -J test/*.test.js test/*/*.test.js",
-    "unit:report": "tap -J test/*.test.js test/*/*.test.js --cov --coverage-report=html --coverage-report=cobertura | tee out.tap",
+    "lint:typescript": "standard --parser @typescript-eslint/parser --plugin @typescript-eslint/eslint-plugin test/types/*.ts fastify.d.ts",
+    "unit": "tap --no-esm -J test/*.test.js test/*/*.test.js",
+    "unit:report": "tap --no-esm -J test/*.test.js test/*/*.test.js --cov --coverage-report=html --coverage-report=cobertura | tee out.tap",
     "unit:junit": "tap-mocha-reporter xunit < out.tap > test/junit-testresults.xml",
     "test:typescript": "tsd",
     "test:report": "npm run lint && npm run unit:report && npm run test:typescript",
     "test": "npm run lint && npm run unit && npm run test:typescript",
     "coverage": "npm run unit -- --cov --coverage-report=html",
-    "coveralls": "npm run unit -- --cov",
+    "test:ci": "npm run lint && npm run unit -- --cov --coverage-report=lcovonly && npm run typescript",
```

This needs to updated to:
```suggestion
    "test:ci": "npm run lint && npm run unit -- --cov --coverage-report=lcovonly && npm run test:typescript",
```